tox.ini

```diff
@@ -5,26 +5,22 @@ minversion = 1.8
 whitelist_externals=sphinx-build
 envlist =
     flake8-py27,
-    flake8-py35,
+    flake8-py36,
     readme-py27,
-    py33-dj18
-    py{27,34,35,py}-dj{18,19,master}
+    py{27,34,35,36,py}-dj{18,110,111}
+    py{34,35,36}-dj{20,master}
 
 [testenv]
-basepython =
-    py27: python2.7
-    py33: python3.3
-    py34: python3.4
-    py35: python3.5
-    pypy: pypy
 usedevelop = true
 setenv =
     DJANGO_SETTINGS_MODULE = tests.settings.main
     DJANGO_CONFIGURATION = Test
 deps =
     -rtests/requirements.txt
-    dj18: https://github.com/django/django/archive/stable/1.8.x.tar.gz#egg=django
-    dj19: https://github.com/django/django/archive/stable/1.9.x.tar.gz#egg=django
+    dj18: django>=1.8,<1.9
+    dj110: django>=1.10,<1.11
+    dj111: django>=1.11,<2.0
+    dj20: https://github.com/django/django/archive/stable/2.0.x.tar.gz#egg=django
```

Comment:
Could also use PyPI here for dj20?!

Reply:
djnago 2.0 dropped python2

Reply:
??
I've meant to use `dj20: django>=2.0a1,<2.1` here.